src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,9 @@
 package racingcar;
 
+import racingcar.controller.RacingGameController;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        RacingGameController.run();
```

Comment:
저도 이번에 새로 알았는데 Console.close() 메서드가 존재하더라고요! 이 메서드를 이용해서 스트림을 닫아 메모리 누수를 방지하는 것도 좋은 방법일 것 같습니다!
```suggestion
        RacingGameController.run();
        Console.close();
```

Reply:
오 제안해 주신 방법 좋은 것 같아요! 감사드립니다 👍🏻